aiida/engine/processes/process.py

```diff
@@ -255,6 +299,13 @@ def kill(self, msg=None):
 
     @override
     def out(self, output_port, value=None):
+        """Register output port.
```

Comment:
If you don't mind a final change (I see you still need to update the branch anyway), I would really rephrase this. Here we are not registering a 'port' but we are registering an actual output node. It just happens to be validated against a certain output port, that was registered in the process spec of the process. So I would suggest writing: "Register an output for the given output port". You could add as a note for clarification "The name of the port will be used as the link label".

Reply:
ah sorry, I hadn't properly read your comment.
I went for "attach output to output port" now.